src/app/Pages/Dashboard/Dashboard.tsx

```diff
@@ -44,78 +51,146 @@ export default function Dashboard(): JSX.Element {
     setPressed(newPressed)
   }
 
-  function handleFretOffset(direction: boolean) {
+  function handleFretOffset(
+    direction: boolean,
+    event: React.MouseEvent<HTMLButtonElement>
+  ) {
+    event.preventDefault()
     let newFretOffset = fretOffset
     if (direction && newFretOffset >= 1) newFretOffset--
     if (!direction && newFretOffset <= 15) newFretOffset++
     setFretOffset(newFretOffset)
   }
 
+  function isStringPressed() {
+    if (Object.values(pressed).find((string) => string > 0)) return true
+    return false
+  }
+
   function handleSafe(event: React.FormEvent<HTMLFormElement>) {
-    const newSavedChords = [pressed, ...savedChords]
-    setSavedChords(newSavedChords)
+    event.preventDefault()
+    if (chordInput && isStringPressed()) {
+      const newChord = {
+        chord: pressed,
+        name: chordInput,
+        offset: fretOffset,
+      }
+      const newSavedChords = [newChord, ...savedChords]
+      setSavedChords(newSavedChords)
```

Comment:
you could use the prev functionality from useState: 
```suggestion
      setSavedChords( prev => [newChord, ...prev])
```